factory/ideas.php

```diff
@@ -416,6 +444,25 @@ public function set_title($idea_id, $title)
 		return true;
 	}
 
+	/**
+	 * Get the title of an idea.
+	 *
+	 * @param int $id ID of an idea
+	 *
+	 * @return string The idea's title
+	 */
+	public function get_title($id)
+	{
+		$sql = 'SELECT idea_title
+			FROM ' . $this->table_ideas . '
+			WHERE idea_id = ' . (int) $id;
+		$result = $this->db->sql_query_limit($sql, 1);
```

Comment:
Can we add some caching here? Maybe 300 or 600 seconds (5/10 minutes)? Not sure what's a good caching time for this.

Reply:
Is that worth it? because a cache purge action will then need to be added as well whenever soemone edits a title.

Reply:
Maybe not. Table probably won't be huge. We can revisit this if we run into performance issues.